jax/_src/pallas/core.py

```diff
@@ -274,6 +266,7 @@ class BlockMapping:
   block_shape: tuple[Mapped | int, ...]
   block_aval: AbstractMemoryRef   # The block ref aval
   index_map_jaxpr: jax_core.ClosedJaxpr
+  index_map_src_info: str  # function_name at filename:linenumber
```

Comment:
Should we introduce a type alias

```
SrcInfo = str  # <name> at <filename>:<line>
```

as a type alias to avoid repeating the comment in multiple signatures?

Reply:
Good idea, I have done this, also for OriginStr.